packages/components/Button/src/Button.tsx

```diff
@@ -69,22 +73,26 @@ export const Button = compose<IButtonType>({
 
     // We shouldn't have to specify the source prop on Slots.icon, here, but we need another drop from @uifabricshared
     return (
-      <Slots.root>
-        <Slots.stack>
-          {info.icon && <Slots.icon source={renderData.slotProps!.icon.source} />}
-          {info.content && <Slots.content />}
-          {children}
-        </Slots.stack>
-      </Slots.root>
+      <Slots.touchable>
```

Comment:
We should be able to replace one of the view layers with the touchable, which is itself a view.

Reply:
So I would change the root to be the one that is sometimes a touchable. I also might try updating the experimental/Button which will replace this one shortly. In the new framework you don't have to have anything called root.

Reply:
Actually, the [TouchableNativeFeedback](https://reactnative.dev/docs/0.5/touchablenativefeedback) must be a wrapper of a single View -- the TouchableNativeFeedback view itself doesn't have a style prop.   I guess I could make the top-most view be TouchableWithoutFeedback for non-Android plats, and TouchableNativeFeedback for Android only.